viewing-data.html

```diff
@@ -432,15 +432,40 @@
 
 <p>Expand the <span class="bold">Screen</span> folder, expand the <span class="bold">Plate</span> container and select a <span class="bold">Run</span>.</p>
 
-<p>Click on a well in the plate shown on the centre pane to view the data in that well.</p>
+<p>Click on a <span class="bold">Well</span> in the Plate shown in the centre pane to view the data in the Well. Well annotations, but not Image annotations, are displayed in right-hand pane. Preview for Wells is not currently available.</p>
 
-<p>Double-click on a well to open the image in the full viewer.</p>
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/viewingDataSPWell.jpg" class="align-center" src="images/viewingDataSPWell.jpg" style="width: 100%;" />
+
+<li><p class="first">Select an <span class="bold">Image</span> (a Field contained in the selected Well) in the lower centre pane.</p>
+
+<p>Annotations for the Image are displayed in right-hand pane. The spatial position of the selected Field is highlighted in the lower left-hand pane.</p>
 
 <div class="line-block">
 <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
 
-<img alt="../images/viewingDataIDR.jpg" class="align-center" src="images/viewingDataIDR.jpg" style="width: 100%;" />
+<img alt="../images/viewingDataSPWImage.jpg" class="align-center" src="images/viewingDataSPWImage.jpg" style="width: 100%;" />
+
+<li><p class="first">Select <span class="bold">Preview</span> tab in right-hand pane to see the preview of the selected Image.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/viewingDataSPWImagePreview.jpg" class="align-center" src="images/viewingDataSPWImagePreview.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<div class="note">
+<p class="note-title">Note</p>
+<p class="note-text">In the Preview panel you can Save new rendering settings to the current Image or 'Save to All' Images in the Plate. In the latter case thumbnails will be updated for currently visible Images in the plate but a browser refresh may be needed to refresh thumbnails from other Fields.</p>
```

Comment:
I would leave may be (because it will not be always needed, e.g. in Safari, if you did not view the fields previously etc.)